workers/unity/Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uxml

```diff
@@ -1,19 +1,47 @@
-<UXML xmlns="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements">
+<UXML xmlns="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xmlns:ns="Improbable.Gdk.Debug.NetStats">
     <uie:ToolbarMenu name="worldSelector" class="world-selector">
         <Style path="Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uss" />
     </uie:ToolbarMenu>
-    <VisualElement name="updates" class="tab">
+    <uie:Toolbar name="tabSelectorContainer" class="tab-selector-container">
+        <Style path="Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uss" />
+        <uie:ToolbarButton name="updateSelector" text="Updates" class="tab-toggle" />
+        <uie:ToolbarButton name="commandSelector" text="Commands" class="tab-toggle" />
+        <uie:ToolbarButton name="worldCommandSelector" text="World Commands" class="tab-toggle" />
+    </uie:Toolbar>
+    <ns:NetStatsUpdatesTab name="updatesTab" class="tab">
+        <Style path="Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uss" />
+        <VisualElement name="headerBar" class="header">
```

Comment:
Nit: Some duplication here that we could factor out if we wanted to:

```
        <Style path="Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uss" />
        <VisualElement name="headerBar" class="header">
            <uie:ToolbarButton text="Command" name="ComponentButton" class="header-column header-column-name" />
            <uie:ToolbarButton text="Op/s in" name="OpsIn" class="header-column" />
            <uie:ToolbarButton text="KB/s in" name="SizeIn" class="header-column" />
            <uie:ToolbarButton text="Op/s out" name="OpsOut" class="header-column" />
            <uie:ToolbarButton text="KB/s out" name="SizeOut" class="header-column" />
            <VisualElement style="min-width: 15px;" />
        </VisualElement>
        <ScrollView name="container" class="updates-view" focusable="false" show-vertical-scroller="true" />
```

Is in every tab declaration. Could be made into its own re-usable element

Reply:
Done